plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Core/Instance/Eq.hs

```diff
@@ -1,25 +1,45 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
+{-# LANGUAGE FlexibleInstances    #-}
 {-# LANGUAGE TypeOperators        #-}
 {-# LANGUAGE UndecidableInstances #-}
 
-module UntypedPlutusCore.Core.Instance.Eq where
+module UntypedPlutusCore.Core.Instance.Eq () where
 
 import           PlutusPrelude
 
 import           UntypedPlutusCore.Core.Type
 
+import           PlutusCore.DeBruijn
 import           PlutusCore.Eq
 import           PlutusCore.Name
 import           PlutusCore.Rename.Monad
 
 import           Universe
 
-instance (GEq uni, Closed uni, uni `Everywhere` Eq, Eq fun, HasUnique name TermUnique) =>
-            Eq (Term name uni fun ann) where
+instance (GEq uni, Closed uni, uni `Everywhere` Eq, Eq fun) =>
+            Eq (Term Name uni fun ann) where
     term1 == term2 = runEqRename $ eqTermM term1 term2
 
+-- I do all this because I want to avoid:
```

Comment:
perhaps this wordwrap newtype can be de-deduplicated in a submodule for both tplc and uplc